src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -236,7 +236,11 @@
         </p>
       </header>
       <TransactionalFolioNumber
-        :validate="getValidateSteps"
+        :folioNumber="getFolioNumber"
+        :transactionalFolioNumber="getTransactionalFolioNumber"
+        :doValidate="getValidateSteps"
+        @update:transactionalFolioNumber="setTransactionalFolioNumber($event)"
+        @valid="setTransactionalFolioNumberValidity($event)"
```

Comment:
I took the getters/setters out of the component so it can be easily moved to the shared components repo.